app/models/shipit/stack.rb

```diff
@@ -239,7 +239,7 @@ def status
       :default
     end
 
-    def lock_reverted_commits!
+    def lock_reverted_commits!(user = AnonymousUser.new)
```

Comment:
There's another code path that usees this https://github.com/Shopify/shipit-engine/blob/868398f243c63b6e5e08a07115c29e36d0048ba6/app/jobs/shipit/github_sync_job.rb#L15-L22

I left that one as commits being locked as a result of syncing with GH don't have an obvious lock author.

Reply:
> syncing with GH don't have an obvious lock author.

I would have used the revert author, but both are fine.

Reply:
🤔 Given `appended_commits.any?(&:revert?)`, there could be multiple no?

Reply:
In SyncJob yes, but here: https://github.com/Shopify/shipit-engine/blob/f6b93d8a242962016e0f82e99c2eb223bdc75003/app/models/shipit/stack.rb#L248-L249

It's not ambiguous.